### PR TITLE
feat(judicial-system): End of session bookings

### PR DESCRIPTION
**Pull request by oddsson**
# End of session bookings

Attach a link to issue if relevant

## What

Instead of having a fixed set of custody restrictions that court users can choose from we now have a text area that is prefilled with the requested custody restrictions. 

## Why

This is more flexible for court users.

## Screenshots / Gifs

Screenshot 2022-01-06 at 13 53 45

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] Formatting passes locally with my changes
- [ ] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/components/AccordionItems/CourtRecordAccordionItem/CourtRecordAccordionItem.tsx [hunk 1]
```diff
@@ -34,14 +34,15 @@ const CourtRecordAccordionItem: React.FC<Props> = ({ workingCase }: Props) => {
   const { formatMessage } = useIntl()
 
   const custodyRestrictions = formatCustodyRestrictions(
```
Comment (gudjong) [inline apps/judicial-system/web/src/components/AccordionItems/CourtRecordAccordionItem/CourtRecordAccordionItem.tsx:36]: This is now part of endOfSessionBookings and should be removed.

File: apps/judicial-system/web/src/components/AccordionItems/CourtRecordAccordionItem/CourtRecordAccordionItem.tsx [hunk 2]
```diff
@@ -34,14 +34,15 @@ const CourtRecordAccordionItem: React.FC<Props> = ({ workingCase }: Props) => {
...
-    workingCase.accusedGender,
-    workingCase.custodyRestrictions,
+    workingCase.requestedCustodyRestrictions,
+    workingCase.isCustodyIsolation,
+    true,
   )
 
-  const alternativeTravelBanRestrictions = formatAlternativeTravelBanRestrictions(
+  const alternativeTravelBanRestrictions = formatTravelBanRestrictions(
```
Comment (gudjong) [inline apps/judicial-system/web/src/components/AccordionItems/CourtRecordAccordionItem/CourtRecordAccordionItem.tsx:42]: Same as above

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx [hunk 3]
```diff
@@ -216,7 +216,7 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
               // Custody restrictions
               isAcceptingCaseDecision(workingCase.decision) &&
                 workingCase.type === CaseType.CUSTODY &&
-                workingCase.custodyRestrictions
+                workingCase.requestedCustodyRestrictions
                   ?.filter((restriction) =>
                     [
                       CaseCustodyRestrictions.ISOLATION,
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx:222]: Isolation should now depend on isCustodyIsolation, which the judge sets.

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx [hunk 4]
```diff
@@ -240,14 +240,11 @@ const SignedVerdictOverviewForm: React.FC<Props> = (props) => {
...
             {
-              // Alternative travel ban restrictions
-              (workingCase.decision ===
-                CaseDecision.ACCEPTING_ALTERNATIVE_TRAVEL_BAN ||
-                (workingCase.type === CaseType.TRAVEL_BAN &&
-                  (workingCase.decision === CaseDecision.ACCEPTING ||
-                    workingCase.decision ===
-                      CaseDecision.ACCEPTING_PARTIALLY))) &&
-                workingCase.custodyRestrictions
+              // Travel ban restrictions
```
Comment (gudjong) [inline apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx:243]: Isn't there some inconsistency for custody cases that get an alternative travel ban ruling?

The court record is prefilled with CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_REQUIRE_NOTIFICATION and CaseCustodyRestrictions.ALTERNATIVE_TRAVEL_BAN_CONFISCATE_PASSPORT, but we show nothing here.

**Review by gudjong — CHANGES_REQUESTED** [review]

File: apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx (hunk 4 above)
Comment (oddsson) [reply apps/judicial-system/web/src/routes/Shared/SignedVerdictOverview/SignedVerdictOverviewForm.tsx:243]: I asked about this on Slack and Baldur said we should not show anything here

Screenshot 2022-01-07 at 10 32 25

**Review by gudjong — APPROVED** [review]

**Pull request merged**